refinery/data_set_manager/utils.py

```diff
@@ -1192,3 +1194,94 @@ def get_solr_response_json(node_uuids):
         'data_set_manager'
     )
     return format_solr_response(solr_response)
+
+
+def update_existing_dataset_with_revised_investigation(existing_dataset_uuid,
```

Comment:
The long name is a code smell. This should simply be a method of the DataSet model.